### PR TITLE
fix on utils.has_coroutine function

### PR DESCRIPTION
**Pull request by dutradda**
Fixed coroutinefunction verification when has async def.
   It is necessary because there's no @asyncio.coroutine decorator to validate

Fixes # .

Changes proposed in this pull request:

 -
 -
 -

### CODE REVIEW

File: connexion/utils.py [hunk 1]
```diff
@@ -148,12 +148,20 @@ def loads(self, data):
 def has_coroutine(function, api=None):
```
Comment (jmcs) [inline connexion/utils.py]: Since the logic of this function is getting more complex can you add a docstring explaining it?

Comment (dutradda) [reply connexion/utils.py]: I will do that @jmcs

Comment (dtkav) [reply connexion/utils.py]: hey @dutradda , any chance you can pick this back up?

**Comment by dtkav** [comment]
It looks like this has been broken for several months.
If you want to merge as-is, I'd be happy to go in and comment it.

**Comment by jmcs** [comment]
@dtkav @dutradda I added the docstring. If you are ok with my description I would merge this.

**Comment by dtkav** [comment]
lgtm!

**Comment by jmcs** [comment]
:+1:

**Pull request merged**